### PR TITLE
Featured card new design

### PR DESCRIPTION
**Pull request by wJoenn**
Part of https://github.com/wJoenn/artify/issues/59

Here's the new design
image

Compared to the old one
image

### CODE REVIEW

File: app/assets/stylesheets/components/_card.scss [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    a {
+      align-items: center;
+      background-image: linear-gradient(135deg, #ebc4bc, #cf897a);
+      border-radius: 5px;
+      box-shadow: 0 0 10px rgba(0, 0, 0, 0.3);
+      display: flex;
+      font-size: 0.8rem;
+      gap: 10px;
+      padding: 10px;
+      transform: skew(-20deg);
```
Comment (MarianaCarlota) [inline app/assets/stylesheets/components/_card.scss:142]: I feel there is few constrast between the text color and the background on the button.  Can you try the text color $secondary to see if is more readable?

**Comment by MarianaCarlota** [comment]
I would drop the title a bit more down or start the blue gradient a bit more up so the text doesn't mix so much with the background and it's easier to read

**Review by MarianaCarlota — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/components/_card.scss (hunk 1 above)
Comment (wJoenn) [reply app/assets/stylesheets/components/_card.scss:142]: I tried it and it definitely is more readable.
I wasn't sure about the looks though
image

**Comment by wJoenn** [comment]
https://user-images.githubusercontent.com/75388869/224578768-9b9438a8-543c-48f3-819e-b6c5798ac8bd.mp4

**Comment by MarianaCarlota** [comment]
ok, increasing the text shadow is a good idea :)

**Comment by Glod33** [comment]
Looks cool!

**Review by MarianaCarlota — APPROVED** [review]

**Pull request merged**